rust/stake_neuron_from_cli/setup_and_run.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
+set -e
+
+# Just a random commit to run the example against.
+IC_COMMIT=4b7cde9a0e3b5ad4725e75cbc36ce635be6fa6a8
+
+mkdir -p wasms
+curl -L -f -o wasms/governance-canister.wasm.gz \
+    "https://download.dfinity.systems/ic/$IC_COMMIT/canisters/governance-canister.wasm.gz"
+curl -L -f -o wasms/ledger-canister_notify-method.wasm.gz \
+    "https://download.dfinity.systems/ic/$IC_COMMIT/canisters/ledger-canister_notify-method.wasm.gz"
```

Comment:
if we already provide a URL to the did file in dfx.json, why not use a url for the wasm as well? we could also use the same commit for did and wasm there

Reply:
Done.  I was relying on some code generation for dfx, as I don't really use it.  I didn't realize that was possible, thanks for pointing it out.